database/migrations/default/1738088180607_notes_audit_fields/up.sql

```diff
@@ -0,0 +1,32 @@
+-- use replica mode to disable triggers
+set session_replication_role = replica;
```

Comment:
@mateoclarke here's the fix for the triggers. a trick we've [used in Moped](https://github.com/cityofaustin/atd-moped/blob/main/moped-database/migrations/1707160404494_clear_component_empty_strings/up.sql) a few times.